src/lib/header-bidding/prebid/bid-config.spec.ts

```diff
@@ -624,11 +624,28 @@ describe('triplelift adapter', () => {
 		});
 	});
 
-	test('should return correct triplelift adapter params for mbu', () => {
+	test('should return correct triplelift adapter params for leaderboard, with requests from Aus or NZ', () => {
+		containsLeaderboard.mockReturnValueOnce(true);
+		containsMpu.mockReturnValueOnce(false);
+		containsDmpu.mockReturnValueOnce(false);
+		containsMobileSticky.mockReturnValueOnce(false);
+		isInAuOrNz.mockReturnValue(true);
+		const tripleLiftBids = bids(
+			'dfp-ad--top-above-nav',
+			[createAdSize(728, 90)],
+			mockPageTargeting,
+		)[1].params;
+		expect(tripleLiftBids).toEqual({
+			inventoryCode: 'theguardian_topbanner_728x90_prebid_AU',
+		});
+	});
+
+	test('should return correct triplelift adapter params, with requests from US or Canada', () => {
```

Comment:
this is for MPU as well right?

```suggestion
	test('should return correct triplelift adapter params for MPU, with requests from US or Canada', () => {
```